packages/rainbowkit/src/components/Avatar/Avatar.tsx

```diff
@@ -47,7 +51,7 @@ export function Avatar({ address, imageUrl, loading, size }: AvatarProps) {
         }}
         userSelect="none"
       >
-        {imageUrl ? (
+        {imageUrl && !overrideEnsImage ? (
```

Comment:
Since the avatar component is passed an ENS image and can decide how to manage it, do we need to include this flag? I feel like alternating between the custom avatar and the ENS image should be managed within the avatar component and this would help reduce complexity here - happy to chat about this if I'm missing something though.

Reply:
originally i did not have `overrideEnsImage` and simply did the `avatar={CustomAvatar}` pattern, but as I was building the little custom avatar for the demo it felt annoying having to "if then" the ENS image and add the custom css to make it fit when we already handle that image nicely ourselves. so my thought was that most developers would want to respect the ENS image of their users, while providing a custom "generative" component like our emojis for those who do not have an ENS image. devs can optionally pass `overrideEnsImage={true}` if they want to fully eject and handle all cases

happy to remove this and let devs handle the image themselves, just stating my original reasoning